app/views/dashboards/pro.html.erb

```diff
@@ -90,6 +90,8 @@
           data-total-readinglist="<%= @dashboard.this_week_reactions.total_by_type_per_day(type: "Reaction", category: "readinglist") %>">
         </canvas>
       </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"
```

Comment:
Is this just to use the ajax API? I think we should go with native `fetch`.

Reply:
fetch doesn't work with internet explorer -- the reason I used this pattern was because it's in the code base a few other times this way. I can totally change to use Axios or Fetch with polyfill though.